### ISSUE TITLE
detectron2 install failed

### ISSUE THREAD

**Issue by shihfangchen** [facebookresearch/meshrcnn#13]
HI when the program install detectron2(pip install -e .)
It returns these:

Collecting detectron2 (from meshrcnn==1.0)
ERROR: Could not find a version that satisfies the requirement detectron2 (from meshrcnn==1.0) (from versions: none)
ERROR: No matching distribution found for detectron2 (from meshrcnn==1.0)

Does anyone have a solution?

Both ubuntu 16.04 and anaconda in windows have same problem.

**Comment by gkioxari** [comment]
Please install detectron2 following the detectron2 installation instructions [here](https://github.com/facebookresearch/detectron2/blob/master/INSTALL.md). This codebase will not install detectron2 for you. Same goes for PyTorch3D.

**Comment by shihfangchen** [comment]
Thanks, I built them on colab and it succeeded!

However, when it runs Mesh R-CNN on my own input image:

!python demo/demo.py \
--config-file configs/pix3d/meshrcnn_R50_FPN.yaml \
--input images/input.jpg \
--output output_demo \
--onlyhighest MODEL.WEIGHTS meshrcnn_R50.pth

it returns:
![image](https://user-images.githubusercontent.com/14999043/75222616-99ed2300-57df-11ea-80b3-be2c0a4aedb0.jpg)

what may the solution of that?

**Comment by NavneetSajwan** [comment]
Still doesn't work. I am trying to install the CPU version. Same error persists

**Comment by chudomozg** [comment]
I have the same problem. I've tried to install cpu version in pycharm.

**Comment by chudomozg** [comment]
> I have the same problem. I've tried to install cpu version in pycharm.

So, If I open link for cpu from install guide https://dl.fbaipublicfiles.com/detectron2/wheels/cpu/torch1.7/index.html
I see only linux distr's, I think it's error explanation. 
I use Windows 10 + Pycharm

**Comment by Ivvvvvy** [comment]
I tried "pip install git+https://github.com/facebookresearch/detectron2" and it worked.

**Comment by patchy631** [comment]
I'm trying to install detectron 2 on a cpu linux machine.
Using official guideline:
python -m pip install detectron2 -f \
  https://dl.fbaipublicfiles.com/detectron2/wheels/cpu/torch1.10/index.html
  
And still facing this error :
Pip subprocess error:
ERROR: Could not find a version that satisfies the requirement detectron2 (from versions: none)

**Comment by rjacobs914** [comment]
I am having this issue as well

> I'm trying to install detectron 2 on a cpu linux machine. Using official guideline: python -m pip install detectron2 -f https://dl.fbaipublicfiles.com/detectron2/wheels/cpu/torch1.10/index.html
> 
> And still facing this error : Pip subprocess error: ERROR: Could not find a version that satisfies the requirement detectron2 (from versions: none)

**Comment by satpalsr** [comment]
I am having the same issue.
```
Looking in links: https://dl.fbaipublicfiles.com/detectron2/wheels/cpu/torch1.9/index.html
ERROR: Could not find a version that satisfies the requirement detectron2 (from versions: none)
ERROR: No matching distribution found for detectron2
```

**Comment by ali20211** [comment]
I am using google colab and I have the same problem:

torch:  1.12 ; cuda:  cu113
Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
Looking in links: https://dl.fbaipublicfiles.com/detectron2/wheels/cu113/torch1.12/index.html
ERROR: Could not find a version that satisfies the requirement detectron2 (from versions: none)
ERROR: No matching distribution found for detectron2

Anyone can help, please

**Comment by xloem** [comment]
Note: at https://dl.fbaipublicfiles.com/detectron2/wheels/cu113/torch1.10/index.html one can see there are versions marked cp36, cp37, cp38, and cp39. This means they are only available for python 3.6, 3.7, 3.8, and 3.9 . If you have python 3.10 there is no available package.

**Comment by Rania123ran** [comment]
> I am using google colab and I have the same problem:
> 
> torch: 1.12 ; cuda: cu113 Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/ Looking in links: https://dl.fbaipublicfiles.com/detectron2/wheels/cu113/torch1.12/index.html ERROR: Could not find a version that satisfies the requirement detectron2 (from versions: none) ERROR: No matching distribution found for detectron2
> 
> Anyone can help, please

hey did you find the solution ?
